specification.markdown

```diff
@@ -386,14 +386,13 @@ The path defines the allowed storage locations of the EBML Element within an EBM
 The path attribute is REQUIRED.
 
 ```
-EBMLFullPath          = [EBMLParentPath] EBMLElementPath
-EBMLParentPath        = EBMLFixedParent EBMLLastParent
-EBMLFixedParent       = *(EBMLPathAtom)
+EBMLFullPath          = [EBMLParentPath] [EBMLGlobalParent] 
+                        EBMLElementPath
```

Comment:
You understand correctly.

Reply:
OK, so IIUC an EBMLFullPath can be an EBMLParentPath followed by an EBMLElementPath with no EBMLGlobalParent, but in that case how could the boundary between the EBMLParentPath and the EBMLElementPath be determined?

Or is EBMLParentPath only relevant to elements with an EBMLGlobalParent? If so, then  EBMLParentPath could be added to EBMLGlobalParent rather than a component of EBMLFullPath.

Reply:
EBMLParentPath is empty or multiple `\abcdefg` parts
EBMLElementPath is non-empty and a `\abcdefg` part.
The EBMLElementPath is the last `\abcdefg` of a full path. The rest is the EBMLParentPath. That's how it is deterministic.

The EBMLGlobalParent is always in the form `(x*y)` which is distinct enough from a EBMLPathAtomRecursive which is in the form `(\abcdef)`.

Void/CRC-32 still include the `\` in the EBMLGlobalParent part. I think it makes it easier to read that it's the repetition of some path, so I'll add it back in EBMLGlobalParent.